util/gh-pages/style.css

```diff
@@ -1,13 +1,60 @@
+* {
```

Comment:
I assume those rules are just copied from the bootstrap files? Or is that your own CSS magic?

Reply:
My magic mostly. ^^'